observability/cockpit/concepts.mdx

```diff
@@ -0,0 +1,174 @@
+
+## Contact points
+
+Contact points define how your contacts are notified when an alert fires. Contact point types include emails, Slack and texts. When an alert fires, a notification is sent to all contact point types listed for a contact point.
+
+</Concept>
+
+<Concept>
+
+## Endpoints
+
+An endpoint is the point of entry in a communication channel when two systems are interacting. The endpoint is the means from which the API can access the resources they need from a server to perform their task. An endpoint can include a URL of a server or service.
+
+</Concept>
+
+<Concept>
+
+## Loki Remote Write
+
+``Loki Remote Write`` is a feature that allows you to send metrics from almost any device to a Loki server.
+
+</Concept>
+
+<Concept>
+
+## LogQL
+
+LogQL is Grafana Loki’s language for querying logs. LogQL uses labels and operators for filtering.
+
+</Concept>
+
+<Concept>
+
+## Logs
+
+Logs provide a record of all events and errors that take place during the lifecycle of your resources. They represent an excellent source of visibility if you want to know when a problem occurred, or which events correlate with it.
+
+</Concept>
+
+<Concept>
+
+## Managed alerts
+
+Managed alerts are alert criteria that Scaleway defines for you. They allow you to receive alerts for behaviors that we deem abnormal on your products.
+
+</Concept>
+
+<Concept>
+
+## Managed dashboards
+
+A managed dashboard is a set of one or more panels organized into one or more rows where you can visualize your data.
+
+</Concept>
+
+<Concept>
+
+## Metric
+
+A metric is a lifecycle-related numerical representation of data (e.g. disk usage and CPU usage) measured over intervals of time. Metrics give you a bird's eye view of your infrastructure.
+
+</Concept>
+
+<Concept>
+
+## Monitoring
+
+Monitoring is a tool that allows you to watch and understand the state of your systems. Monitoring is based on gathering predefined sets of metrics or logs.
+
+</Concept>
+
+<Concept>
+
+## Prometheus Remote Write
+
+``Prometheus Remote Write`` is a feature that allows you to send your metrics from almost any device to a Prometheus server.
+
+</Concept>
+
+<Concept>
+
+## PromQL
+
+PromQL, short for Prometheus Querying Language, is the main way to query metrics within Prometheus. It is designed for building queries for graphs and alerts.
+
+</Concept>
+
+<Concept>
+
+## Receivers
+
+Receivers are hubs consisting of contacts points. You can associate one or several alerts with one or more receivers. This allows you to diversify your alerts.
+
+</Concept>
+
+<Concept>
+
+## Recording rules
+
+Recording rules allow you to precompute frequently needed or computationally expensive expressions and save their results as a new set of time series.
+
+</Concept>
+
+<Concept>
+
+## Samples
+
+A sample is a unique measuring point on a time series.
+
+</Concept>
+
+<Concept>
+
+## Time series
+
+A time series is a sequence of numerical data points in successive order, that tracks a sample over time.
```

Comment:
A time series is a sequence of numerical data points in successive order, that tracks an information over time.

(the sample is a point in the time series, as such the time series cannot track it 😄)

Reply:
Agreed "information" is not an excellent term here, couldn't come with a better one